Pythonwin/stdafx.h

```diff
@@ -45,7 +45,9 @@ programmers who forget to use the new names. */
 #include "traceback.h"
 #include "pythonrun.h"
 #include "import.h"  // Python: for dynamicattach routines.
+#if PY_VERSION_HEX < 0x030A0000
```

Comment:
I think you can unconditionally kill this.

Reply:
Done / trying.